BeatTogether.DedicatedServer.Kernel/Managers/LobbyManager.cs

```diff
@@ -147,194 +181,15 @@ public void Update()
                             }, DeliveryMethod.ReliableOrdered);
                     }
                 }
-
                 _lastEntitlementState = allPlayersOwnBeatmap;
 
-                // Figure out if should be counting down and for how long
-                switch (_configuration.SongSelectionMode)
+                switch (_configuration.SongSelectionMode) //server modes
                 {
```

Comment:
Combined code that runs mananger selects lobbies and quickplay lobbies to reduce discrepancies between the two as they both essentially work the same way.